### DISCUSSION TITLE
WebView2 in Hololens

### DISCUSSION THREAD

**Discussion by hzhong16** [MicrosoftEdge/WebView2Feedback#3401] [Q&A] [answered]
I notice that WebView2 is push out in Hololens 2 on insider preview. I Had update my Hololens 2 to the newest Dev Version but I am not sure where to begin with implement it in to Hololens, there is not much documentation on it. 

Can anyone give me a direction where should i start? I am currently developing my Hololens application inside the unity.

Thank you

**Comment by champnic** [comment, marked as answer]
Hey @hzhong16 - Docs have just been published:
https://learn.microsoft.com/en-us/windows/mixed-reality/develop/advanced-concepts/webview2
API Ref: https://learn.microsoft.com/en-us/windows/mixed-reality/develop/advanced-concepts/webview2-unity-plugin
Tutorial: https://learn.microsoft.com/en-us/microsoft-edge/webview2/get-started/hololens2
